.evergreen/config.yml

```diff
@@ -1852,3 +1935,9 @@ buildvariants:
     display_name: "API Version ${os-ssl-32}"
     tasks:
       - name: ".versioned-api"
+
+  - matrix_name: "kms-tls-test"
+    matrix_spec: { version: ["4.2", "4.4", "latest"], os-ssl-32: ["ubuntu1604-64-go-1-15"] }
```

Comment:
Do we have to run this against all server versions for which CSFLE is supported? If the tests exist to assert KMS behavior, which doesn't depend on server version, what benefit do we get from doing this?

Reply:
A good question... I _think_ you're right that as long we are testing against a single >= 4.2 version we should be covered. @kevinAlbs does running only on `latest` make sense?

Reply:
I agree, there is no benefit to testing on more than one server version. Just testing against `latest` sounds good to me!